actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -108,7 +112,9 @@ private void courseProgressMetricsReport(Request actorMessage) {
         ElasticSearchUtil.getDataByIdentifier(
             EsIndex.sunbird.getIndexName(), EsType.course.getTypeName(), batchId);
     if (isNull(courseBatchResult) || courseBatchResult.size() == 0) {
-      ProjectLogger.log("CourseMetricsActor-courseProgressMetrics-- batch is not valid .");
+      ProjectLogger.log(
+          "CourseMetricsActor:courseProgressMetricsReport  course is not valid.",
```

Comment:
From:
"CourseMetricsActor:courseProgressMetricsReport  course is not valid."

To:
"CourseMetricsActor:courseProgressMetricsReport: batchId not found."